test/utils.js

```diff
@@ -1,128 +0,0 @@
-/* Copyright 2020 Record Replay Inc. */
-
-const fs = require("fs");
-const os = require("os");
-const https = require("https");
-const { spawnSync } = require("child_process");
-
-const GeckoInstallDirectories = [
-  "/Applications/Replay.app", // macOS
-  `${process.env.HOME}/replay`, // linux
-  "C:\\Program Files\\replay", // windows
-];
-
-const GeckoSuffixes = [
-  "Contents/MacOS/replay", // macOS
-  "Contents/MacOS/firefox", // macOS unbranded
-  "replay", // linux, using distribution.
-  "firefox", // linux unbranded
-  "dist/bin/replay", // linux, testing with local build.
-  "dist/bin/firefox", // linux local unbranded
-  "replay.exe", // windows
-];
-
-// Get the executable to use when starting gecko.
-function findGeckoPath() {
```

Comment:
The backend uses this function, see test/runlive/run.ts.  I think this logic should be copied to the backend before removing it here, so the dependency on the devtools can be removed.